prepare/tasks/generation.py

```diff
@@ -3,7 +3,7 @@
 
 add_to_catalog(
     FormTask(
-        inputs=["input"],
+        inputs=["input", "type_of_input"],
```

Comment:
Maybe would be also good to have type of output?
```suggestion
        inputs=["input", "type_of_input", "type_of_output"],
```

Reply:
Done